src/main/java/InputData/newPlaylistInputData.java

```diff
@@ -0,0 +1,47 @@
+package InputData;
+import Entities.Song;
+import Entities.User;
+import java.util.Optional;
+
+public class newPlaylistInputData {
+    private final String playlistName;
+    private final User owner;
+    private final Song firstSong;
+    private boolean privacy;
```

Comment:
Could we rename this to be isPrivate? I think that would be a better name since it's a boolean variable and it would also be more consistent with the naming convention you used in your other classes (since you previously used isPublic)

Reply:
Ok, I will update that.